plugins/chat/assets/javascripts/discourse/components/chat-drawer/header/right-actions.js

```diff
@@ -3,5 +3,10 @@ import { inject as service } from "@ember/service";
 
 export default class ChatDrawerHeaderRightActions extends Component {
   @service chat;
+  @service router;
   @service chatStateManager;
```

Comment:
I also think this one might not be used anymore (although not due to your PR)

Reply:
oh yeah removing